src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -690,7 +693,7 @@ const TimeEntryForm = props => {
                   name="isTangible"
                   checked={formValues.isTangible}
                   onChange={handleInputChange}
-                  disabled={!canEditTimeEntry && !initialIsTangible}
+                  disabled={(!canEditTimeEntry && !initialIsTangible || canEditTimeEntryDescription)}
```

Comment:
```suggestion
                  disabled={(!canEditTimeEntry && !initialIsTangible && !canEditTimeEntryDescription)}
```
Does this need to be inversed?